.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,26 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["**"]
+
+permissions: {}
+
+jobs:
+  zizmor:
+    name: Run zizmor 🌈
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+      contents: read # only needed for private repos
+      actions: read # only needed for private repos
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
+
+      - name: Run zizmor 🌈
+        uses: zizmorcore/zizmor-action@e673c3917a1aef3c65c972347ed84ccd013ecda4 # v0.2.0
```

Comment:
I suppose this is technically more secure than a soft pin. How are we going to make sure this stays up to date?

Reply:
I believe there a dependabot setting that can keep these up to date as well?

Reply:
Great! Thanks for checking that Ryan 👍

Reply:
For what it's worth, I've seen reports that it's not able to update the `# v4` comment when it should (when v5 comes out)